index.js

```diff
@@ -980,6 +980,54 @@
   //. ```
   S.init = S.slice(0, -1);
 
+  //# take :: Number -> [a] | String -> Maybe [a] | Maybe String
```

Comment:
> Should we mention it anywhere in the docs that that is how we treat strings?

The thing is we don't support strings everywhere we support arrays. `S.slice` supports both types because `slice` is defined for both types. `S.take` and `S.drop` can support both since they can be defined in terms of `slice`.

The approach I favour is:

  - using a clear signature (even if it doesn't tell the full story);
  - stating in the docstring that strings are supported (as you have done); and
  - including an example of applying `S.take(n)` to a string (as you have done).